docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -557,6 +558,11 @@ This key can be listed multiple times.
 Set the log-driver used by Podman when running the container.
 Equivalent to the Podman `--log-driver` option.
 
+### `LogOpt=`
+
+Set the log-opt (logging options) used by Podman when running the container.
+Equivalent to the Podman `--log-opt` option.
```

Comment:
Please add `This key can be listed multiple times.`

Reply:
Added